modules/special-rules/pages/damaging-flyers.adoc

```diff
@@ -0,0 +1,19 @@
+= Damaging flyers
+
+Successful rolls against airborne flyers work differently to other units -- we use an extra step to determine what actually happens.
+
+---
+
+Each time you damage a Flyer unit, roll another D6:
+
+* If the result scores equal to or greater than the Flyer unit's Armour value then the Flyer suffers a hit.
+If this destroys the unit then remove it as normal for a destroyed unit, otherwise it is *driven off*.
+* If the result scores less than the Flyer unit's Armour value then the flyer is not destroyed -- but it is temporarily damaged, and is *driven off*.
```

Comment:
Again, we can't use 'being hit', because that's the biggest cause of problems with both the original GW text and your first rewrite. If you don't like 'damage' we could use 'has a successful hit die allocated to it', but that's likely to get very wordy.

Mirroring the bullets sounds good, but I think it would be better to follow 'damage' with 'the damage is only temporary' or similar.

```suggestion
* If the result scores equal to or greater than the Flyer unit's Armour value then it suffers a hit.
If this destroys the unit then remove it as normal for a destroyed unit, otherwise it is *driven off*.
* If the result scores less than the Flyer unit's Armour value then the flyer does not suffer a hit -- the damage is only temporary and it is *driven off*.
```